### DISCUSSION TITLE
DeepL translator support

### DISCUSSION THREAD

**Discussion by bluenitepk** [SubtitleEdit/subtitleedit#5085] [Q&A]
Congratulations on a great program to the author.
I use it to edit subtitles for my personal, home use only. No commercial use whatsoever.
I want to ask if there is a possibility of DeepL translator support? I know it's not easy to link, but I find DeepL much better than Gtranslate. I repeat, for home use only, no API key.
Thank you for the responses
Peter

**Comment by niksedk** [comment]
Hi Peter, you can use the "Auto-translate via copy paste..."

**Reply by bluenitepk** [reply]
perfect. GTrans mangles text and characters. Thanks a lot. You are the best of all subtitle editors. Fingers crossed
